azext_iot/deviceupdate/providers/base.py

```diff
@@ -0,0 +1,190 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# flake8: noqa: E402
+
+from azext_iot.deviceupdate.providers.loaders import reload_modules
+reload_modules()
```

Comment:
why is this needed?

Reply:
Added description as a docstring to the `reload_modules` function. I'm planning on further fleshing this out with conditional loading or potential alternative way to get the same effect.